src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html

```diff
@@ -353,21 +353,51 @@ <h3>Path Configuration</h3>
                        data-bind="textInput: dataPath, attr: { placeholder: $root.databaseModel.name() || 'Enter database path (optional)' }"
                        title="Enter location for the database files."/>
                 
-                <ul class="dropdown-menu" role="menu"
-                    data-bind="autoComplete: '#dataPath', foreach: $root.recentPathsAutocomplete.createCompleter()">
-                    <li role="presentation" data-bind="click: _.partial($root.recentPathsAutocomplete.complete, $data)">
+                <ul class="dropdown-menu max-height" role="menu"
+                    data-bind="autoComplete: '#dataPath', foreach: $root.allAutoCompleteOptions">
+                    <li role="presentation" data-bind="click: $data.isRecent ? _.partial($root.recentPathsAutocomplete.complete, $data.path) : $parent.dataPath.bind($parent, $data.path)">
                         <a role="menuitem" tabindex="-1" href="#">
-                            <span data-bind="text: $data"></span>
+                            <!-- ko if: $data.isRecent -->
+                            <strong data-bind="text: $data.path"></strong>
+                            <!-- /ko -->
+                            <!-- ko ifnot: $data.isRecent -->
+                            <span data-bind="text: $data.path"></span>
+                            <!-- /ko -->
                         </a>
                     </li>
                 </ul>
             </div>
         </div>
         <div class="form-group row">
-            <div class="col-sm-12">
+            <div class="col-sm-12" data-bind="if: $root.databaseLocationInfoMessage">
                 <p class="text-info bg-info padding padding-sm">
-                    The data files will be created in:<br/>
-                    <strong><small data-bind="text: $root.databaseLocationShowing"></small></strong>
+                    <span data-bind="text: $root.databaseLocationInfoMessage"></span><br />
+                    <div data-bind="if: $root.spinners.databaseLocationInfoLoading">
+                        <span class="btn-spinner"></span>
```

Comment:
for inline spinner you should use class: 'global-spinner' IIRC

Reply:
done